third_party/txt/src/txt/line_metrics.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LIB_TXT_SRC_LINE_METRICS_H_
+#define LIB_TXT_SRC_LINE_METRICS_H_
+
+#include <map>
+#include <vector>
+
+#include "run_metrics.h"
+
+namespace txt {
+
+class LineMetrics {
```

Comment:
Please zero out all members. Not all of them are set in the constructor and will end up containing garbage values.